### PR TITLE
Add service key variable to auth

### PR DESCRIPTION
**Pull request by kewne**

### CODE REVIEW

File: src/@orb.yml [hunk 1]
```diff
@@ -158,6 +158,7 @@ jobs:
     steps:
       - checkout
       - gcr-auth:
+          gcloud-service-key: <<parameters.gcloud-service-ke>>
```
Comment (marc-howard-ovo) [inline src/@orb.yml]: Missing "y" for the word "key"

Comment (marc-howard-ovo) [inline src/@orb.yml]:
```suggestion
          gcloud-service-key: <<parameters.gcloud-service-key>>
```

**Comment by kewne** [comment]
@marc-howard-ovo Would you like for me to squash the commits?

**Comment by ymaki** [comment]
Hi, I faced the same problem and found this PR.
I think we should add `gcloud-service-key: <<parameters.gcloud-service-key>>` to L55, gcp-cli/initialize step, to make it works, shouldn't we?

**Comment by iynere** [comment]
apologies for the long wait here—https://github.com/CircleCI-Public/gcp-gcr-orb/pull/14 will fix this

**Pull request closed without merging**